qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -267,6 +267,9 @@ def run(
         # pylint: disable=arguments-differ
         if job_share_level:
             try:
+                warnings.warn("The `job_share_level` keyword is deprecated due to low adoption "
+                              "and will be removed in a future release.",
+                              DeprecationWarning, stacklevel=3)
```

Comment:
Sorry if I wasn't clear in my previous comment. Normally we'd deprecate something for 3 months before removing it. But in this case job share level already doesn't work, so there's really no deprecating it (deprecation implies it still works). My comment was that we should keep the parameter / method in place, so user's existing code doesn't blow up (not that anyone is using lol). If the parameter/method is referenced, we can issue a warning saying the feature is no longer supported by IBM Quantum, and the parameter/method itself will also be removed in a future release. There is also no point passing it to api_client since it already doesn't work on the server side.

Reply:
@jyu00 got it, lol, I have updated now, please review.